include/dsn/c/api_layer1.h

```diff
@@ -536,7 +536,9 @@ typedef union dsn_msg_context_t
         uint64_t is_request : 1;           ///< whether the RPC message is a request or response
         uint64_t is_forwarded : 1;         ///< whether the msg is forwarded or not
         uint64_t is_replication_needed: 1; ///< whether state replication is needed for this request
-        uint64_t unused : 8;               ///< not used yet
+        uint64_t is_response_in_piece: 1;  ///< set when a response is marshalled in different place
```

Comment:
@shengofsun any more detailed comments about these flags? the current comments are not easy to follow.